src/pages/costModels/createCostModelWizard/markup.tsx

```diff
@@ -1,79 +1,115 @@
 import {
   FormGroup,
+  Grid,
+  GridItem,
   InputGroup,
   InputGroupText,
+  List,
+  ListItem,
+  Radio,
   Stack,
   StackItem,
   Text,
   TextContent,
   TextInput,
-  TextVariants,
   Title,
 } from '@patternfly/react-core';
 import { Form } from 'components/forms/form';
 import React from 'react';
 import { WithTranslation, withTranslation } from 'react-i18next';
 
+import { styles } from '../costModel/costCalc.styles';
 import { CostModelContext } from './context';
 
-interface MarkupValidationState {
-  isValid: boolean;
-}
-
-class Markup extends React.Component<WithTranslation, MarkupValidationState> {
-  public state = {
-    isValid: true,
-  };
-
+class Markup extends React.Component<WithTranslation> {
   public render() {
     const { t } = this.props;
-    const { isValid } = this.state;
 
     return (
       <CostModelContext.Consumer>
-        {({ onMarkupChange, markup }) => {
+        {({ handleSignChange, handleOnKeyDown, handleMarkupDiscountChange, markupValidator, markup, isDiscount }) => {
           return (
             <Stack hasGutter>
               <StackItem>
                 <Title headingLevel="h2" size="xl">
-                  {t('cost_models_wizard.markup.title')}
+                  {t('cost_calculations')}
                 </Title>
               </StackItem>
               <StackItem>
+                <Title headingLevel="h3" size="md">
+                  {t('cost_models_details.markup_or_discount')}
+                </Title>
                 <TextContent>
-                  <Text component={TextVariants.h6}>{t('cost_models_wizard.markup.sub_title')}</Text>
+                  <Text>{t('cost_models_wizard.description_markup_or_discount_model')}</Text>
```

Comment:
The reason we use `Text` is to define the `h6` tag for screen readers. We shouldn't eliminate the variant so the text isn't bold. It's still represents a new section that screen readers need to announce to users.

I know it probably matches the mocks, but it's necessary for a11y.

Reply:
Done